packages/trader/src/Modules/CFD/Helpers/constants.ts

```diff
@@ -6,7 +6,7 @@ const DEMO_DXTRADE_URL = 'https://dx-demo.deriv.com/';
 const DXTRADE_IOS_APP_URL = 'https://apps.apple.com/us/app/deriv-x/id1563337503';
 const DXTRADE_ANDROID_APP_URL = 'https://play.google.com/store/apps/details?id=com.deriv.dx';
 
-const getServerName = is_demo => (is_demo ? 'Deriv-Demo' : 'Deriv-Server');
+const getServerName = (is_demo: 'Deriv-Demo') => (is_demo ? 'Deriv-Demo' : 'Deriv-Server');
```

Comment:
Why does it have to be `Deriv-Demo`?

Reply:
I'll make is_demo optional in another commit. According to the condition of the function, If we want to mention that we have the demo server we need the parameter with a specific value ('Deriv_Demo'), otherwise we leave the parameter empty to get the else condition value.